esphome/const.py

```diff
@@ -502,6 +505,7 @@
 UNIT_HZ = 'hz'
 UNIT_HECTOPASCAL = 'hPa'
 UNIT_KELVIN = 'K'
+UNIT_KILOGRAMS = 'kg'
```

Comment:
```suggestion
UNIT_KILOGRAM = 'kg'
```

The unit names are all singular.